src/components/atoms/NavigationLogo.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<p class="navigation__logo">{{text}}</p>
+</template>
+
+<script>
+	import json from "../../../data/structure.json"
+	export default {
+		name: "navigationLogo",
+		data() {
+			return {
+				text: json.navigation.logo
+			}
+		}
+	}
+</script>
+
+<style lang="scss" scoped>
+	.navigation__logo {
+		font-size: 25px;
+		letter-spacing: 3px;
+		padding: 0 0 0 10px;
```

Comment:
Is it required to set top right and bottom padding to 0?

Reply:
I was suggested to use universal padding notation and avoid using `padding-right/left/top/bottom`